customer-commons/src/test/java/no/unit/nva/customer/model/ChannelConstraintPolicyTest.java

```diff
@@ -6,22 +6,22 @@
 import static java.lang.String.format;
 import static java.util.Arrays.stream;
 import static java.util.stream.Collectors.joining;
-import static no.unit.nva.customer.model.ChannelConstraint.DELIMITER;
-import static no.unit.nva.customer.model.ChannelConstraint.ERROR_MESSAGE_TEMPLATE;
+import static no.unit.nva.customer.model.ChannelConstraintPolicy.DELIMITER;
+import static no.unit.nva.customer.model.ChannelConstraintPolicy.ERROR_MESSAGE_TEMPLATE;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.core.IsEqual.equalTo;
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
-class ChannelConstraintTest {
+class ChannelConstraintPolicyTest {
     public static final String NONSENSE = "nonsense";
 
     @Test
     void shouldThrowRuntimeExceptionWhenInputIsInvalid() {
-        Executable executable = () -> ChannelConstraint.lookUp(NONSENSE);
+        Executable executable = () -> ChannelConstraintPolicy.lookUp(NONSENSE);
         var exception = assertThrows(RuntimeException.class, executable);
-        var expected = format(ERROR_MESSAGE_TEMPLATE, NONSENSE, stream(ChannelConstraint.values())
-                .map(ChannelConstraint::toString).collect(joining(DELIMITER)));
+        var expected = format(ERROR_MESSAGE_TEMPLATE, NONSENSE, stream(ChannelConstraintPolicy.values())
```

Comment:
Always tempting, but just type the string again…if the template string mutates, then the test will still pass, which isn't an expectation.